package.json

```diff
@@ -3,7 +3,7 @@
   "version": "2.2.0",
   "description": "Node.js bindings for libddwaf",
   "main": "index.js",
-  "libddwaf_version": "1.7.0",
+  "libddwaf_version": "1.8.0",
```

Comment:
From Anil:
>Hello, I've just released a small patch for the [WAF (v1.8.1)](https://github.com/DataDog/libddwaf/releases/tag/1.8.1) with some fixes for invalid rulesets, there are no API changes so please use this version instead of v1.8.0.
```suggestion
  "libddwaf_version": "1.8.1",
```

Reply:
The only change is:
>Return NULL handle when incorrect version or empty rules provided to ddwaf_init (https://github.com/DataDog/libddwaf/pull/139)

Reply:
Updated